docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -872,13 +872,13 @@ when the string is `NULL` or it doesn't contain valid JSON, or the JSON value
 is not an array.
 
 ```sql title="Examples"
-JSON_ARRAY_LENGTH("[1, 2, 3]") => 3
-JSON_ARRAY_LENGTH("[1, [1, [2]], 3]") =>  3
-JSON_ARRAY_LENGTH("[]") => 0
-JSON_ARRAY_LENGTH("{}") => NULL
-JSON_ARRAY_LENGTH("123") => NULL
+JSON_ARRAY_LENGTH('[1, 2, 3]') => 3
+JSON_ARRAY_LENGTH('[1, [1, [2]], 3]') =>  3
+JSON_ARRAY_LENGTH('[]') => 0
+JSON_ARRAY_LENGTH('{}') => NULL
+JSON_ARRAY_LENGTH('123') => NULL
 JSON_ARRAY_LENGTH(NULL) => NULL
-JSON_ARRAY_LENGTH("abc") => throws an "Invalid JSON format" exception
+JSON_ARRAY_LENGTH('abc') => throws an "Invalid JSON format" exception
```

Comment:
This appears to return `null` in 0.24.0.

Reply:
When a UDF throws an exception, it returns `null` and logs the error message in server logs.

I see how this can be unclear from this page. WDYT about replacing `throws an "Invalid JSON format" exception` with `returns NULL and logs "Invalid JSON format" exception in server logs`?